homework_2/task1.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+
+int recFib(int num)
+{
+    if (num <= 1)
+    {
+        return 1;
+    }
+    return recFib(num - 2 ) + recFib(num - 1);
+}
+int iterFib(int num)
+{
+    int last = 1;
+    int first = 1;
+    for(int i = 2; i <= num; i++)
+    {
+        int temp = first;
+        first = first + last;
+        last = temp;
+    }
+    return first;
+
+}
+
+int main() 
+{
+    int num = 0;
+    
+    printf("This program finds fibonacci number\n");
```

Comment:
```suggestion
    printf("This program finds Fibonacci number\n");
```
Текст должен быть грамотным, даже если его печатает программа